docs/backend-api/getting-started.md

```diff
@@ -213,9 +213,9 @@ Session hash can be obtained via `user/auth` API call:
 
 === "GET"
     This method is not recommended. Just for example:
-
+    ```abap
     [api_base_url]/user/auth?login=demo&password=demo

```

Comment:
I propose to do the same as in the panel API: two tabs `POST` and `GET` with cURL on both.

Reply:
for those who will use some program like the postman, it will be much more convenient to use this option for copying - in one line the entire request in the form "/user/ auth?login=demo&password=demo. In the program itself, you can choose what the request will be - get or post. I tried to copy cURL requests there - it turns out not at all convenient. Two tabs - great idea, but I suggest using one tab for POST cURL and one tab for GET in one line for other systems like postman

Reply:
Sounds reasonable